localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,43 +1,346 @@
 ﻿l_english:
+ ####################################################
+ # Special titles
+ ### Places ###
+ city_ahn_qiraj:0 "City"
+ quarter_grim_batol:0 "Quarter"
+ city_grim_batol:0 "City"
```

Comment:
I would rather make holdings cities than this. It may confuse the player.

Reply:
deleted in ac0afe713